polaris-react/src/components/Filters/Filters.tsx

```diff
@@ -97,8 +97,6 @@ export interface FiltersProps {
   onAddFilterClick?: () => void;
   /** Whether the filter should close when clicking inside another Popover. */
   closeOnChildOverlayClick?: boolean;
-  /** The name of the currently selected view */
-  selectedViewName?: string;
```

Comment:
Is this a breaking change? Could we deprecate the prop?

Reply:
We have a v13 branch. So if you want to remove it all together we can do it in that branch. If you want to get folks to stop using it a deprecation and warning would help a ton!

Reply:
We want to get this out ASAP due to merchant frustration with the current experience, so will go with deprecating!